assets/js/base/components/product-price/index.tsx

```diff
@@ -239,6 +239,10 @@ export interface ProductPriceProps {
 	 * Custom style to apply to the regular price wrapper.
 	 */
 	regularPriceStyle?: React.CSSProperties | undefined;
+	/**
+	 * Custom margin to apply to the price wrapper.
+	 */
+	spacingStyle?: React.CSSProperties | undefined;
```

Comment:
`React.CSSProperties` includes all possible CSS Properties. While `spacingStyle` should include only spacing related ones, right? Like `margin` and `padding` only? Or is it only `margin`?

Reply:
Right. For now it's just `margin`.